src/backends/torch/torchlib.cc

```diff
@@ -1530,13 +1578,165 @@ namespace dd
         ad_res.add("clnames", clnames);
         ad_res.add("nclasses", nclasses);
       }
+    if (_bbox)
+      {
+        ad_res.add("bbox", true);
+        ad_res.add("pos_count", entry_id);
+        ad_res.add("0", ad_bbox);
+      }
     ad_res.add("batch_size",
                entry_id); // here batch_size = tested entries count
     SupervisedOutput::measure(ad_res, ad_out, out, test_id, test_name);
     _module.train();
     return 0;
   }
 
+  double area(const std::vector<double> &bbox)
+  {
```

Comment:
maybe move below functions out of torchlib.cc?

Reply:
Do you have a suggestion for where it could be moved? I think these function can apply to all backends, so maybe in `src/utils`?

Reply:
ok for src/utils :)

Reply:
you can also create `src/bbox_utils` files, it is a common practice as bboxes require some specific ops.

Reply:
So `src/utils/bbox.hpp`?